release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -297,16 +297,32 @@ For more information, see xref:../networking/metallb/metallb-configure-bgp-peers
+With this update, you can associate a Virtual Routing and Forwarding (VRF) instance with a network interface by using a `NodeNetworkConfigurationPolicy` custom resource. By associating a VRF instance with a network interface, you can support traffic isolation, independent routing decisions, and the logical separation of network resources. This is available as a Technology Preview feature.
 
 For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc#virt-example-host-vrf_k8s_nmstate-updating-node-network-config[Example: Network interface with a VRF instance node network configuration policy].
 
 [id="ocp-414-broadcom-bcm57504-support"]
 ==== Support for Broadcom BCM57504 is now GA
 
-Support for the Broadcom BCM57504 network interface controller is now available for the SR-IOV Network Operator. 
+Support for the Broadcom BCM57504 network interface controller is now available for the SR-IOV Network Operator.
 
-For more information, see xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported devices].
+For more information, see xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov
+
+[Supported devices].
+[id="ocp-4-14-creating-subinterface"]
+==== MAC-VLAN, IP-VLAN, and VLAN subinterface creation for pods
+
+With this release, the ability to create a MAC-VLAN, IP-VLAN and VLAN subinterface based on a master interface in a container namespace is generally available. This feature allows you to create the master interfaces as part of the pod network configuration in a separate network attachment definition, and base the VLAN, MACVLAN or IPVLAN on this interface without requiring the knowledge of the network configuration of the node. For more information, see xref:../networking/multiple_networks/configuring-additional-network.html#nw-about-configuring-master-interface-container_configuring-additional-network[About configuring the master interface in the container network namespace].
+
+[id="ocp-4-14-tap-device-plugin"]
+==== Enhance network flexibility by using the TAP device plugin
+
+This release introduces a new Container Network Interface (CNI) network plugin type: the tap device plugin. This plugin allows the creation of tap devices within containers. A tap device allows user-space programs to handle network frames, acting as an interface that receives frames from and sends frames to user-space applications instead of through traditional network interfaces. For more information, see xref:../networking/multiple_networks/configuring-additional-network.html#nw-multus-tap-object_configuring-additional-network[Configuration for a TAP additional network].
```

Comment:
Is it `TAP` or `tap`? Heading and paragraph use difference capitalization schemes.

Reply:
Going to go with TAP strands for  "Terminal Access Point."